### PR TITLE
Add global prettier configuration and fix formatting

### PR DESCRIPTION
**Pull request by michalinacienciala**
We added repo-wide prettier configuration to handle all possible file
types.
With lint:js we are supporting `*.js` files.
With lint:sol we are supporting `*.sol` files.

We were missing handling for bunch of other file types, like `*.json`,
`*.sh`, `*.yaml` which prettier allows us to.

As part of this PR we're also fixing formatting errors reported by
prettier/linter.

Closes: #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "scripts": {
+    "format": "prettier --check .",
+    "format:fix": "prettier --write ."
+  },
+  "devDependencies": {
```
Comment (michalinacienciala) [inline package.json:6]: I'm not sure weather I should also add eslint-related dependencies and scripts in this package.json file.

**Comment by michalinacienciala** [comment]
I didn't create a `.prettierignore` file, as I didn't know if there will be anything to ignore. At this point I don't think there's anything, but in the future we may have something related to the @lukasz-zimnoch's yearn work.

**Comment by dimpar** [comment]
Don't we need to remove "prettier" from [.eslintrc](https://github.com/keep-network/tbtc-v2/blob/global-format/solidity/.eslintrc#L2)?

**Comment by michalinacienciala** [comment]
> Don't we need to remove "prettier" from [.eslintrc](https://github.com/keep-network/tbtc-v2/blob/global-format/solidity/.eslintrc#L2)?
> 
> The rest lgtm

You're right. I was inspired by https://github.com/keep-network/keep-ecdsa/blob/main/solidity/.eslintrc, but now realised that this file needs the same change that you suggested. I removed the `prettier` reference in cee5d696a308506922302d3ad2a790ac8d60eb21 and in `keep-ecdsa` in https://github.com/keep-network/keep-ecdsa/pull/867.

**Review by dimpar — DISMISSED** [review]

**Comment by dimpar** [comment]
Latest changes also look good to me. @nkuba do you want to take a look at this PR as well?

**Comment by nkuba** [comment]
This PR allows us to tweak a little bit the configuration in `solidity` and `yearn` packages. I did it in https://github.com/keep-network/tbtc-v2/pull/21. We should merge both PRs together.

**Comment by dimpar** [comment]
@nkuba / @michalinacienciala  there was a conflict `solidity/package.json` with the `main` that I've resolved. All looks good to me, but since I've merged the branch + resolved the confilct, can you pls take a look at the `solidity/package.json`, give a green light if looks good to you, and I'll merge this guy.

**Comment by michalinacienciala** [comment]
> @nkuba / @michalinacienciala there was a conflict `solidity/package.json` with the `main` that I've resolved. All looks good to me, but since I've merged the branch + resolved the confilct, can you pls take a look at the `solidity/package.json`, give a green light if looks good to you, and I'll merge this guy.

Looks ok! 🚀

**Comment by michalinacienciala** [comment]
@nkuba, could you check the settings of jobs required for merging of PRs to main? Currently we require `contracts-lint`, but we've renamed this job to `contracts-format`.

**Review by nkuba — APPROVED** [review]

**Pull request merged**